subsys/net/l2/wifi/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config WIFI_FEAT_RAW_SCAN_RESULTS
+	bool "Raw scan results"
+	help
+	  This option enables raw scan results. When enabled, the raw scan
+	  results (beacons or probe responses) are provided to the application.
```

Comment:
Please mention here that enabling this config will increase the net_mgmt event size considerably.